pyartcd/pyartcd/pipelines/build_microshift.py

```diff
@@ -116,15 +118,13 @@ async def run(self):
             if assembly_type is not AssemblyTypes.STREAM and not self.force:
                 nvrs = await self._find_builds()
 
+            if assembly_type is not AssemblyTypes.STREAM:
+                await self.slack_say(f":construction: Microshift prep for assembly {self.assembly} :construction:")
+
             if nvrs:
                 self._logger.info("Builds already exist: %s", nvrs)
             else:
-                # Rebases and builds microshift
-                if assembly_type is not AssemblyTypes.STREAM:
-                    slack_client = self.runtime.new_slack_client()
-                    slack_client.bind_channel(self.group)
-                    slack_response = await slack_client.say(f":construction: Build microshift for assembly {self.assembly} :construction:")
```

Comment:
Sometimes we were creating slack_thread and sometime not, other times we were relying on slack_thread to indicate something else (like if assembly is stream). So simplify this branching, always create a top level slack message for named assemblies for each run